DrinkManagerWeb/Views/Shared/_Layout.cshtml

```diff
@@ -8,41 +8,54 @@
-                <button class="navbar-toggler" type="button" data-toggle="collapse" data-target=".navbar-collapse" aria-controls="navbarSupportedContent"
-                        aria-expanded="false" aria-label="Toggle navigation">
-                    <span class="navbar-toggler-icon"></span>
-                </button>
-                <div class="navbar-collapse collapse d-sm-inline-flex flex-sm-row-reverse">
-                    <ul class="navbar-nav flex-grow-1">
-                        <li class="nav-item">
-                            <a class="nav-link text-dark" asp-area="" asp-controller="Home" asp-action="Index">Home</a>
-                        </li>
-                        <li class="nav-item">
-                            <a class="nav-link text-dark" asp-area="" asp-controller="Home" asp-action="Privacy">Privacy</a>
-                        </li>
-                    </ul>
-                </div>
+    <!-- sidebar -->
+    <div class="container-fluid">
+        <div class="row">
+            <div class="col-2 sidenav">
+                <h2 class="website-name"> Drink Manager</h2>
+                <ul class="navbar-nav flex-grow-1 sidebar-list">
+                    <li class="nav-item">
+                        <a class="nav-link text-dark" asp-area="" asp-controller="Drinks" asp-action="Index">All drinks</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link text-dark" asp-area="" asp-controller="Home" asp-action="Index">Search by name</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link text-dark" asp-area="" asp-controller="Home" asp-action="Privacy">Search by alcohol content</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link text-dark" asp-area="" asp-controller="Home" asp-action="Privacy">Search by ingredient</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link text-dark" asp-area="" asp-controller="Home" asp-action="Privacy">Favourite drinks</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link text-dark" asp-area="" asp-controller="Home" asp-action="Privacy">Reviewed drinks</a>
+                    </li>
+                </ul>
             </div>
-        </nav>
-    </header>
-    <div class="container">
-        <main role="main" class="pb-3">
-            @RenderBody()
-        </main>
-    </div>
 
-    <footer class="border-top footer text-muted">
-        <div class="container">
-            &copy; 2020 - DrinkManagerWeb - <a asp-area="" asp-controller="Home" asp-action="Privacy">Privacy</a>
+            <div class="col-10 main-body">
+
+                <!-- page body -->
+                <div>
+                    @RenderBody()
+                </div>
```

Comment:
This div element does not do anything, we could remove it.

Reply:
I would consider leaving @RenderBody() with its div. RenderBody is a placeholder where all the page-specific views show up, wrapped in the layout page.